frontend/src/libs/components/app/app.tsx

```diff
@@ -49,6 +49,9 @@ const App: React.FC = () => {
         <>
           <h2>Users:</h2>
           <h3>Status: {dataStatus}</h3>
+          <div style={{ width: '500px', height: '500px' }}>
```

Comment:
I know about it. This div should not be there yet, I added it only for an example of how the map will look like. I can either remove it from there, or I can put it as a style module (but I don't know where to add it yet, because the map will be used on most pages of the application, but it can have different sizes)

Reply:
got it, thanks